src/stream.ts

```diff
@@ -205,7 +205,7 @@ export class DataAndMoneyStream extends Duplex {
    *
    * This promise will only resolve when the absolute amount specified is reached, so lowering the `sendMax` may cause this not to resolve.
    */
-  async sendTotal (limit: BigNumber.Value): Promise<void> {
+  async sendTotal (limit: BigNumber.Value, timeout = 60000): Promise<void> {
```

Comment:
I think maybe this should be an options object as the second arg, because something like `sendTotal(1000, 1000)` isn't as clear as `sendTotal(1000, { timeout: 1000 })`